subjects/rt/README.md

```diff
@@ -0,0 +1,107 @@
+## rt
+
+There are two ways to render a 3d scene into a 2d image: `rasterization` that basically converts the shapes and geometric to pixels and applies calculations to obtain the color, shadows, refraction, etc... of those pixels. The other method is called `ray tracing` and consists in drawing each pixel with already its color, shadow, refraction, reflection, etc....
+
+Here we can see a visual difference between each method:
+
+![image.png](raytrace.png)
+
+![image.png](raytrace2.png)
```

Comment:
I think now you should delete those two. The other one looks much better anyway haha

Reply:
ahah thanks, done